internal/metrics.go

```diff
@@ -16,10 +16,11 @@ func MakeMetrics(namespace, subsystem string) (*kitprometheus.Counter, *kitprome
 		Help:      "Number of requests received.",
 	}, []string{"method"})
 	latency := kitprometheus.NewSummaryFrom(stdprometheus.SummaryOpts{
-		Namespace: namespace,
-		Subsystem: subsystem,
-		Name:      "request_latency_microseconds",
-		Help:      "Total duration of requests in microseconds.",
+		Namespace:  namespace,
+		Subsystem:  subsystem,
+		Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
```

Comment:
I would prefer leaving comment that explains this magic numbers. Not everybody who reads the code know exactly Prometheus API/terminology without checking the reference - comment will save them some time.